src/dodal/beamlines/i19_2.py

```diff
@@ -0,0 +1,65 @@
+from dodal.common.beamlines.beamline_utils import (
+    device_factory,
+)
+from dodal.common.beamlines.beamline_utils import (
+    set_beamline as set_utils_beamline,
+)
+from dodal.devices.hutch_shutter import HutchShutter
+from dodal.devices.synchrotron import Synchrotron
+from dodal.devices.zebra.zebra import Zebra
+from dodal.devices.zebra.zebra_constants_mapping import (
+    ZebraMapping,
+    ZebraSources,
+    ZebraTTLOutputs,
+)
+from dodal.log import set_beamline as set_log_beamline
+from dodal.utils import BeamlinePrefix
+
+# NOTE all workstations on I19 default to i19-1 as beamline name
+# Unless variable is exported (which is not usually done by scientists)
+# NOTE All PVs for both hutches have the prefix BL19I
+BL = "i19-2"
+PREFIX = BeamlinePrefix(BL)
```

Comment:
Should: It's actually a bug that `BeamlinePrefix(i19-2) == BL19I` (https://github.com/DiamondLightSource/dodal/issues/926). So I would change this here and in `i19_1.py` to be explicit i.e. `PREFIX = BeamlinePrefix("i19", "I")`